src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import core.basesyntax.utils.DataUtils;
+
 public class SalaryInfo {
+    static final String FORMAT = "dd.MM.yyyy";
+    private final StringBuilder report = new StringBuilder();
```

Comment:
make local variable, wont work when class will be called twice
```suggestion
    private final StringBuilder report = new StringBuilder();
```

Reply:
thank you, updated